spec/ruby/core/thread/inline_rescue_spec.rb

```diff
@@ -0,0 +1,9 @@
+require File.expand_path('../../../shared/kernel/raise', __FILE__)
+
+describe "inline rescue" do
```

Comment:
Ah, this exists: https://github.com/rubinius/rubinius/blob/master/spec/ruby/language/variables_spec.rb#L4

Since this file is already pretty big I'd suggest putting the specs in `spec/ruby/language/optional_assignments_spec.rb`, otherwise they can be added as an extra `describe` block in this existing file.